### ISSUE TITLE
Kotlin Multiplatform support

### ISSUE THREAD

**Issue by friendoye** [mplatvoet/kovenant#12]
Is there any plan to make Kovenant library multiplatform?

**Comment by mplatvoet** [comment]
Yes, because the whole API is already detached from the implementation and kotlin 1.2 has multiplatform support.
